app/views/atlases/show.html.haml

```diff
@@ -1,47 +1,86 @@
-- content_for :extrahead do
-  = stylesheet_link_tag "atlases/show"
-  = javascript_include_tag "https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"
-  = javascript_include_tag "https://d3js.org/d3.v3.min.js"
-  = javascript_include_tag "https://d3js.org/d3.geo.projection.v0.min.js"
-  = javascript_include_tag "https://d3js.org/topojson.v1.min.js"
+- content_for :extrajs do
   = javascript_include_tag "atlases/show"
-  = javascript_tag "world = #{@atlas_place_json.to_json.html_safe};"
-  = javascript_tag "taxon_id = #{@atlas.taxon_id}"
+  = javascript_include_tag "atlases/map"
+
+- content_for :extracss do
+  = stylesheet_link_tag "atlases/show"
+  = stylesheet_link_tag "atlases/map"
 
 .container
   .row
     .col-xs-12
+      - if @num_obs > 0
+        .alert.alert-warning{:role => "alert"}
```

Comment:
We are preferring in new code to use the newer Ruby hash syntax `{ role: "alert" }` in ruby and HAML files (or alternatively with a symbol as the value `{ role: :alert }` if applicable). It's easier to type, and is easier when moving back and forth between Ruby and JS. There are several cases of this in these views. Not a huge deal nor a blocker, but another for the "style guideline"